bedrock/src/primitives/config.rs

```diff
@@ -48,6 +48,13 @@ impl BedrockEnvironment {
     }
 }
 
+/// Returns the current `BedrockEnvironment`, defaulting to `Production` if
+/// the global configuration has not been initialized.
+#[must_use]
+pub fn current_environment_or_default() -> BedrockEnvironment {
```

Comment:
I think this could swallow potentially undefined behavior where the config is not globally set before calling Bedrock methods. I think we should at least log an error if the config is not set